addons/website_forum/views/forum_forum_templates.xml

```diff
@@ -188,7 +188,7 @@
 <!-- Edition: ask your question -->
 <template id="new_question" name="New Post">
     <t t-call="website_forum.header">
-        <div t-if="request.env.user.forum_waiting_posts_count" class="alert border" role="alert">
+        <div t-if="not can_post" class="alert border" role="alert">
             <b>You already have a pending post.</b><br/>
```

Comment:
As `can_post` is still reflecting only whether there is a pending post, and we're only limiting the scope of the restriction, what do you think about a `has_pending_post` variable (with the opposite value to the current `can_post`) to replace `request.env.user.forum_waiting_posts_count`?

Also, I think that using a model field for it is kinda neat in Odoo, like it was before.
I'm also adding a kind-of-similar feature for the ability to moderate in https://github.com/odoo/odoo/pull/123709/commits/3b3d9a5a1fdf88f4de59582bb89e441e1e066919#diff-29ba65738803e2d5e8a5751d7048811648cdc4662c46fb68209b103f346104b7R132-R137
 (where it *is* called `can_`moderate ;-)), as a computed field on the forum depending on the context uid (as we do in many places in forum and eLearning). 

I think it would reduce the diff required here if you did the same (and let the ORM do things for us so the value is available in the templates).

Reply:
@flch-odoo okay, thanks for commenting, i will see the pr tomorrow